src/surveyPage.js

```diff
@@ -1,11 +1,15 @@
-import { Wrapper } from "./components/styled";
+import { Wrapper, H1, AccentText, H1Light, H2 } from "./components/styled";
 import { Link } from "react-router-dom";
 import Button from "./components/button/button";
 
 function SurveyPage() {
   return (
     <Wrapper>
-      <h1>this is the survey</h1>
+      <H1 bold>this is the survey</H1>
+      <H1Light>
+        Your <AccentText color={"#719680"}>healing</AccentText> manifestation
```

Comment:
we should add our colors to a colors constants file, we can do this in a future PR though if you want